guacamole-common/src/main/java/org/apache/guacamole/protocol/GuacamoleClientInformation.java

```diff
@@ -144,5 +149,25 @@ public void setOptimalResolution(int optimalResolution) {
     public List<String> getImageMimetypes() {
         return imageMimetypes;
     }
+    
+    /**
+     * Return the timezone as reported by the client.
+     * 
+     * @return
+     *     A string value of the timezone reported by the client.
+     */
+    public String getTimezone() {
+        return timezone;
+    }
+    
+    /**
+     * Set the string value of the timezone.
```

Comment:
Same here: as it will not otherwise be clear to a user of this class, the standard that these timezone strings should conform to should be documented.